README.md

```diff
@@ -1,82 +1,122 @@
-# ONETIME SECRET - v0.13.0-RC1 (2024-04-05)
+# Onetime Secret - v0.13.0
 
-### **Initial release candidate with Ruby 3+ support: [v0.13.0-RC1](https://github.com/onetimesecret/onetimesecret/releases/tag/v0.13.0-RC1).**
+*Keep passwords and other sensitive information out of your inboxes and chat logs.*
 
-_See the [v0.12.0](https://github.com/onetimesecret/onetimesecret/releases/tag/v0.12.0) for the final release with Ruby 2.6.8._
+### Latest releases 
 
----
+* **Ruby 3+: [v0.13.0-RC2](https://github.com/onetimesecret/onetimesecret/releases/tag/v0.13.0-RC2)**
+* Ruby 2.6, 2.7: [v0.12.1](https://github.com/onetimesecret/onetimesecret/releases/tag/v0.12.1)
 
+---
 
-*Keep passwords and other sensitive information out of your inboxes and chat logs.*
 
-## What is a Onetime Secret? ##
+## What is a Onetime Secret?
 
 A one-time secret is a link that can be viewed only once. A single-use URL.
 
-<a class="msg" href="https://onetimesecret.com/">Give it a try!</a>
+Try it out on <a class="msg" href="https://onetimesecret.com/">OnetimeSecret.com</a>!
 
-## Why would I want to use it? ##
+
+## Why would I want to use it?
 
 When you send people sensitive info like passwords and private links via email or chat, there are copies of that information stored in many places. If you use a one-time link instead, the information persists for a single viewing which means it can't be read by someone else later. This allows you to send sensitive information in a safe way knowing it's seen by one person only. Think of it like a self-destructing message.
 
-## How to install
+
+## Installation
```

Comment:
Consider adding a section on "Upgrading" with instructions on how to upgrade to a newer version of Onetime Secret. This could include steps like backing up the config files, pulling the latest Docker image, and any database migrations that may be required. [medium]